### DISCUSSION TITLE
Vram requirement for dreambooth?

### DISCUSSION THREAD

**Discussion by Deathawaits4** [bmaltais/kohya_ss#1818] [Q&A]
Im trying to run a dreambooth training on 24 gigs of vram, but it keeps going OOM. How much is required? Im training adamw8bit and i dont think it should use that much vram.

**Comment by GrigoryPrutskov** [comment]
@Deathawaits4
 I have tesla p100 gpu (with 24gb of vram too) and OOM during training happened periodically. Try to disable caching latents, set gradient accumulator steps to 1, enable gradient checkpoiniting,  use mixed precision with fp16, xformers and memory efficient attention. 
For training sd models these settings utilize ~22-23 gb with adamw8bit, ~13 with adafactor and ~16 with lion8bit.
